fluent-react/package.json

```diff
@@ -44,6 +44,12 @@
   "engines": {
     "node": ">=10.0.0"
   },
+  "jest": {
+    "setupFilesAfterEnv": [
+      "./test/setup.js"
+    ],
+    "restoreMocks": true
```

Comment:
This removes the mock after every run, which helps with reproducibility of tests.